wd_demo.au3

```diff
@@ -236,6 +226,26 @@ Func _RunDemo_GUISwitcher($iState, $idBrowsers, $idDebugging, $idUpdate, $idHead
 	Next
 EndFunc   ;==>_RunDemo_GUISwitcher
 
+Func _RunDemo_Headless($idHeadless)
```

Comment:
I would move the two new functions `Func _RunDemo_Headless` and `Func _RunDemo_Update` above `Func _RunDemo_ErrorHander` to have a better flow of reading the code.

Reply:
Please check: [_RunDemo_* reordering + fixed comments](https://github.com/Danp2/au3WebDriver/pull/193/commits/e343c1c78772c387919c07c29d3c69578b145d91)